docs/reference/contributing/target/itm.md

```diff
@@ -0,0 +1,48 @@
+### Instrumented Trace Macrocell
+
+For targets with Arm CoreSight (for example, Cortex-M3 and Cortex-M4), the Instrumented Trace Macrocell provides a lightweight, nonintrusive way to collect debug trace output.
+
+#### Assumptions
+
+##### Defined behavior
+
+When initialized, writing data to the ITM stimulus registers results in the data being transmitted over the SWO line.
```

Comment:
Query: When what is initialized, by whom or by what?
Query: Who or what transmits the data over the SWO line? The ITM?